src/Ints.php

```diff
@@ -42,4 +44,13 @@ public static function ordinalize(int $number) : string
 
         return "{$number}{$ord}";
     }
+
+    public static function monthName(int $monthNumber) : string
+    {
+        if ($monthNumber < 1 || $monthNumber > 12) {
+            throw new Exception("Invalid month number: the value must be between 1 and 12.");
```

Comment:
Generally, this looks fine, but it's good practice to use an exception class that is as specific as possible. There's a full list of them here: https://www.php.net/manual/en/spl.exceptions.php

Reply:
> It might be useful to compare these two:
> 
> * https://www.php.net/manual/en/class.invalidargumentexception.php
> * https://www.php.net/manual/en/class.unexpectedvalueexception.php

Throwing an exception based on the value does seem better than the type.
I was also considering `OutOfRangeException` but it's related to array indexes.
`UnexpectedValueException` it is!